newdle/newdle.cfg.example

```diff
@@ -89,3 +89,9 @@ EXCHANGE_DOMAIN = ''
 EXCHANGE_PROVIDER_SERVER = ''
 EXCHANGE_PROVIDER_ACCOUNT = ''
 EXCHANGE_PROVIDER_CREDENTIALS = ('', '')
+
+# Cleanup configuration (values are in days)
+# Days after which an incomplete newdle is cleaned up after its last update.
```

Comment:
I'd use "is deleted" here to be clear that this doesn't just "clean something up" but really deletes those newdles